### ISSUE TITLE
Check-coverage and cover has different coverage results

### ISSUE THREAD

**Issue by alicecodes** [gotwarlost/istanbul#583]
I'm using istanbul@1.0.0-alpha.2 and I'm using this command to run my tests: 

`istanbul cover _mocha -- --recursive --compilers js:babel-core/register`

This is the output from istanbul cover:
<img width="813" alt="screen shot 2016-03-30 at 2 04 57 pm" src="https://cloud.githubusercontent.com/assets/1619724/14157576/70cfbbb4-f680-11e5-8a4c-1cb9249a4110.png">

this is the output from istanbul check-coverage: 
<img width="780" alt="screen shot 2016-03-30 at 2 05 29 pm" src="https://cloud.githubusercontent.com/assets/1619724/14157589/7e07a648-f680-11e5-8db5-dc516cbaf73f.png">

Looks like this issue is related: [https://github.com/istanbuljs/istanbul-api/issues/2 ](url)

**Comment by abenhamdine** [comment]
I see exactly the same problem : one of my files is 100% covered with ```istanbul cover``` but 87,5% for branches with ```istanbul check-coverage```

According to my tests spec, the true coverage percentage is 100% for this specific file, so ```istanbul cover``` seems to be correct.

**Comment by gotwarlost** [comment]
Yes, I need to get back to hacking on istanbul and fix this bug. I hope to devote some time to this effort shortly.

**Comment by jlouazel** [comment]
Is there any news on this issue?

**Comment by Jimmy02020** [comment]
any updates ?
